From c40d7642e0a8c301c656301fcf805e3a39b10a94 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 11 Dec 2024 14:52:57 +0800
Subject: [PATCH 237/280] AOSCOS: staging: sb105x: rename state to __state in
 task_struct

Per commit 2f064a59a11f ("sched: Change task_struct::state"), the member
`state' of `struct task_struct' was renamed into `__state'.

Revise driver code accordingly.

Fixes: "AOSCOS: Revert "staging: sb105x: delete the driver""
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/staging/sb105x/sb_pci_mp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/sb105x/sb_pci_mp.c b/drivers/staging/sb105x/sb_pci_mp.c
index e333245ec9c5..963f3091a5c9 100644
--- a/drivers/staging/sb105x/sb_pci_mp.c
+++ b/drivers/staging/sb105x/sb_pci_mp.c
@@ -1051,7 +1051,7 @@ static int mp_wait_modem_status(struct sb_uart_state *state, unsigned long arg)
 		cprev = cnow;
 	}
 
-	current->state = TASK_RUNNING;
+	current->__state = TASK_RUNNING;
 	remove_wait_queue(&state->info->delta_msr_wait, &wait);
 
 	return ret;
-- 
2.48.1

